ViAn/Analysis/analysismethod.cpp

```diff
@@ -209,64 +177,10 @@ void AnalysisMethod::scale_frame() {
 /**
  *  Getters and setters
  */
-
```

Comment:
A lot of code are just moved from this file and motion detection to analysis settings. This pull request is shorter than it looks.